spec/features/monitoring_feature_spec.rb

```diff
@@ -10,6 +10,15 @@
     visit authenticated_root_path
   end
 
+  scenario "It works if there are no minions", js: true do
```

Comment:
I'm not sure I'm following... I got this error when I had no minions available, so this test should already cover that, no ?